dali/pipeline/operators/resize/new_resize.h

```diff
@@ -143,7 +143,7 @@ class NewResize : public Resize<Backend> {
   void SetupSharedSampleParams(Workspace<Backend> *ws) override {
     Resize<Backend> ::SetupSharedSampleParams(ws);
   }
-  virtual uint ResizeInfoNeeded() const     { return t_crop + t_mirrorHor; }
+  virtual uint ResizeInfoNeeded() const override { return t_crop + t_mirrorHor; }
```

Comment:
You don't need virtual if you have override already.

Reply:
Done